modules/rosa-mobb-prereq-checklist.adoc

```diff
@@ -0,0 +1,198 @@
+
+Once you have the above prerequisites installed and enabled, proceed to the next steps.
+
+
+== SCP Prerequisites
+
+ROSA clusters are hosted in an AWS account within an AWS organizational unit. A link:https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_scps.html[service control policy (SCP)] is created and applied to the AWS organizational unit that manages what services the AWS sub-accounts are permitted to access. 
+
+* Ensure that your organization's SCPs are not more restrictive than the roles and policies required by the cluster.
+* Ensure that your SCP is configured to allow the required `aws-marketplace:Subscribe` permission when you choose *Enable ROSA* from the console, and see link:https://docs.aws.amazon.com/ROSA/latest/userguide/troubleshoot-rosa-enablement.html#error-aws-orgs-scp-denies-permissions[AWS Organizations service control policy (SCP) is denying required AWS Marketplace permissions] for more details.
+* When you create a ROSA classic cluster, an associated AWS OpenID Connect (OIDC) identity provider is created. 
+** This OIDC provider configuration relies on a public key that is located in the `us-east-1` AWS region. 
+** Customers with AWS SCPs must allow the use of the `us-east-1` AWS region, even if these clusters are deployed in a different region.
+
+== Networking Prerequisites
+
+Prerequisites needed from a networking standpoint.
+
+=== Firewall
+
+* Configure your firewall to allow access to the domains and ports listed in link:https://docs.openshift.com/rosa/rosa_planning/rosa-sts-aws-prereqs.html#osd-aws-privatelink-firewall-prerequisites_rosa-sts-aws-prereqs[AWS firewall prerequisites].
+
+=== Custom DNS
+
+* If you want to use custom DNS, then the ROSA installer must be able to use VPC DNS with default DHCP options so it can resolve hosts locally. 
+** To do so, run `aws ec2 describe-dhcp-options` and see if the VPC is using VPC Resolver: 
++
+[source,terminal]
+----
+$ aws ec2 describe-dhcp-options
+----
++
+* Otherwise, the upstream DNS will need to forward the cluster scope to this VPC so the cluster can resolve internal IPs and services.
+
+== PrivateLink Prerequisites
+
+If you choose to deploy a PrivateLink cluster, then be sure to deploy the cluster in the pre-existing BYO VPC:
+
+* Create a public and private subnet for each AZ that your cluster uses.
+** Alternatively, implement transit gateway for internet and egress with appropriate routes.
+* The VPC's CIDR block must contain the `Networking.MachineCIDR` range, which is the IP address for cluster machines. 
+** The subnet CIDR blocks must belong to the machine CIDR that you specify.
+* Set both `enableDnsHostnames` and `enableDnsSupport` to `true`.
+** That way, the cluster can use the Route 53 zones that are attached to the VPC to resolve cluster internal DNS records.
+* Verify route tables by running: 
++
+[source,terminal]
+ ----
+ $ aws ec2 describe-route-tables --filters "Name=vpc-id,Values=<vpc-id>"
+ ----
+ +
```

Comment:
This doesn't render correctly.
![image](https://github.com/openshift/openshift-docs/assets/16167833/08b836a0-1c5d-456a-a33f-2158784876a1)

I think you can delete the `+` at 189 and it should work.